src/modules/apps/install.js

```diff
@@ -8,33 +8,44 @@ import {
   wildVersionPattern,
 } from '../../manifest'
 
+const ARGS_START_INDEX = 2
+
+function installApps (apps) {
+  const app = apps.shift() || `${manifest.vendor}.${manifest.name}@${manifest.version}`
+  log.debug('Starting to install app', app)
+  const appRegex = new RegExp(`^${vendorPattern}.${namePattern}@${wildVersionPattern}$`)
+  if (!appRegex.test(app)) {
+    log.error('Invalid app format, please use <vendor>.<name>@<version>')
+    return Promise.resolve()
+  }
+
+  return installApp(app)
+  .then(() => log.info(`Installed app ${app} successfully`))
+  .then(() =>
+    apps.length > 0
+      ? installApps(apps)
+      : Promise.resolve()
+  )
+  .catch(err => {
+    if (err.statusCode === 409) {
+      return log.error(`App ${app} already installed`)
+    }
+    return Promise.reject(err)
```

Comment:
Actually, here's how a fail would look like
![selection_005](https://cloud.githubusercontent.com/assets/5040487/20113897/7a553354-a5d9-11e6-825a-75beba2553df.png)

I believe it's clear enough already.
What do you think?